content/_index.md

```diff
@@ -1,14 +1,14 @@
 ---
-title: Bruiloft Tonke & Jonathan
+title: Welkom lieve gasten
 banner: /img/home.jpg
 ---
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. In vel semper lorem. Nulla neque odio, porttitor tristique suscipit sed, blandit consequat magna. Maecenas consectetur finibus ullamcorper. Aliquam eleifend nisl vel tincidunt pretium. Vestibulum pellentesque, mauris vitae scelerisque tincidunt, felis ipsum elementum mi, ut efficitur ligula nunc id augue. Suspendisse quis lobortis ante, scelerisque imperdiet augue. Aliquam vel metus mollis, vestibulum nisi id, venenatis lectus. Curabitur sed feugiat ante. Suspendisse erat nisl, elementum vitae mi non, facilisis consectetur velit.
+Zaterdag 20 juni 2020 geven Tonke en Jonathan elkaar het ja-woord! Zij doen dat in Burgers' Zoo in Arnhem. Heb je een trouwkaart ontvangen? Meld je dan snel aan/af via het formulier op de [aanmeld](http://localhost:1313/aanmelden/) pagina. 
 
-Quisque sem turpis, condimentum at augue in, commodo auctor leo. Morbi blandit auctor ex. Aenean et tincidunt lectus, placerat maximus augue. Donec vel risus tempus, aliquam purus id, tincidunt dolor. Nullam efficitur tempus neque id lacinia. Cras semper, felis malesuada congue facilisis, mi urna porttitor augue, a maximus magna turpis in est. Cras sit amet vestibulum dui. Morbi sit amet libero id lectus imperdiet ornare sed in elit. Vestibulum justo magna, imperdiet sit amet porta ut, convallis sed odio. Donec auctor est in ante viverra ornare.
+<!-- HIER MOET NOG EEN AANMELDKNOP -->
```

Comment:
Moeten we die nog maken dan? En zo ja,  hoe moet die er dan uitzien?

Reply:
Ja, die moet nog gemaakt worden. Maar wordt pas opgeleverd als we de trouwkaarten gaan sturen. Er is een taak aangemaakt op de backlog